grafana-toolkit: No matching version found for @grafana/e2e-selectors

**What happened**:

```
npx "@grafana/toolkit"@canary plugin:create my-plugin
```

fails with the following error:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @grafana/e2e-selectors@^7.1.0-pre.0-e2c404518b.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of '@grafana/ui'
npm ERR! notarget 
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcus/.npm/_logs/2020-05-08T07_15_34_854Z-debug.log
Install for @grafana/toolkit@canary failed with code 1
```

--- a/scripts/circle-release-next-packages.sh
+++ b/scripts/circle-release-next-packages.sh
@@ -54,7 +54,7 @@ if [ -z "$count" ]; then
 else
   echo "Changes detected in ${count} packages"
   echo "Releasing packages under ${PACKAGE_VERSION}-${GIT_SHA}"
-  npx lerna version "${PACKAGE_VERSION}-${GIT_SHA}" --no-git-tag-version --no-push --force-publish -y
+  npx lerna version "${PACKAGE_VERSION}-${GIT_SHA}" --exact --no-git-tag-version --no-push --force-publish -y
   echo $'\nGit status:'
   git status -s
 

